cms/static/sass/_build-v1.scss

```diff
@@ -69,6 +69,7 @@
 @import 'views/video-upload';
 @import 'views/certificates';
 @import 'views/maintenance';
+@import 'views/course-rerun';
```

Comment:
Move this import file to `_build-v2.scss` as edX is going to deprecate the v1.

Reply:
@hassanarif426 Let's move these changes to v2 and see they are working properly, they don't break anything and they compile perfectly